fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -368,6 +380,42 @@ public FedoraResource getContainer() {
     }
 
     @Override
+    public FedoraResource getLDPCv() {
```

Comment:
This function doesn't seem to be used yet, what was your plan around this?

Reply:
@whikloj I was thinking about in cases like DELETE, HEAD, GET of the TimeMap/LDPCv, we may want to throw a Exception if the TimeMap LDPCv container is not found instead of using FindOrCreated() to create a new LDPCv. What do you think?

Reply:
I think trying to DELETE/HEAD/GET a resource in the Repository that doesn't exist should throw an Exception based on normal interactions. But I could be mistaken on that.

But this function could be called from the parent LDPRv to `get` the TimeMap resource when creating the header links for example. So that's fine.